### DISCUSSION TITLE
Upgrade R-Link (1) to R-Link evolution => one issue with traffic announcements

### DISCUSSION THREAD

**Discussion by moltes** [cedricp/ddt4all#958] [Q&A] [answered]
Hi there,

I recently purshased a R-Link Evolution screen unit from a damaged car, and replaced my current R-Link unit with this one.
All the features are there and working properly (work even better since I updated the soft from 11.336 to 11.347, the previous owner didn't make any soft updates).

The only issue that remains is with "traffic announcement", also known as "TA" or "iTraffic".
If activated in the settings, when an announcement is available, it turns off the sound (radio/sd/maybe bluetooth) and broadcasts that traffic announcement.

In my case, it's desactivated, but every ten seconds (quite annoying), it turns the sound off and very quickly back on (because there is no announcement).
I think it exists a parameter for this feature that is slightly different between the R-Link and R-Link Evolution units, so that the 'new' unit thinks there is a signal but there is not, which I hope could be found through DDT4ALL.

I've searched a bit but couldn't find something related to the traffic annoucements.
Would you know about it, or are you crazy enough to take up the challenge (-: ?

**Comment by AndyIreland** [comment]
would the TMC disabling work on this if you set it to 'disabled' ? 
![image](https://github.com/cedricp/ddt4all/assets/49172217/c322732a-8fdc-482e-a60f-d49dbc08663b)

don't forget to reboot r-link after you have made the change

**Reply by moltes** [reply]
Hi Andy, 

thanks for the idea.
One interesting thing is that I didn't have access to the traffic parameters in the R-Link Settings menu (the button was inactive). In DDT4All, "TMC disabling" was indeed already set to "TMC disabled". I made it enabled to check it anyway, but after rebooting the R-Link, the issue was still there.

**Reply by AndyIreland** [reply]
When you are on the FM Radio , and go into the radio settings on the r-link is there not a setting there to turn off traffic announcements? - I thought there was I saw once

**Reply by AndyIreland** [reply]
I have just been outside and took this picture for you. have you got this box unticked?
![image](https://github.com/cedricp/ddt4all/assets/49172217/38956c8b-2cd7-4621-99ac-7cf635e9db48)

**Reply by moltes** [reply]
There are these options yes, but I already tried to untick them all without success unfortunately

**Reply by AndyIreland** [reply]
ah right - strange then , I am not sure - nothing in the TomTom map settings part to turn it off no?

**Reply by moltes** [reply]
There are a couple of settings to turn on/off the display of traffic information on the map and alert sounds. I tried to play with them but nothing changed.
I'll have a look at accoustic parameters in DDT4All, don't know yet how it looks like.

**Comment by moltes** [comment]
Found another issue, when I mute the sound volume (mute mode), it unmutes itself after a few seconds (when the bug occurs I assume)

**Comment by moltes** [comment, marked as answer]
Finally solved the issue by buying a new radio module (the grey box) from another damaged car that had R-Link Evolution as well

**Reply by AndyIreland** [reply]
and thats cured it has it? - I would have said it would lie in the MFD rather than the radio unit, but if thats fixed it then great

**Reply by moltes** [reply]
I think they've also changed something in the radio module so they both need to be replaced if you want to upgrade your system.
There are several P/N for this module, and each is linked to a "SW" number (SoftWare, who knows..)
281151559R SW 3.3.1.4 for R-Link Evolution
I originally had 281151761R SW 3.0.1.7
